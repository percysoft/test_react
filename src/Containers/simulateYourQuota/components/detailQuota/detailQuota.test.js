import React from 'react';
import '@testing-library/jest-dom';
import { ItemDetail } from './index';
import { shallow } from 'enzyme';

describe('Test Item Detail Component', () => {
  test('Render Item ', () => {
    let title = 'TEA'
    let amount= 10000
    const wrapper = shallow (<ItemDetail amount={amount} title={title}/>)
    expect(wrapper).toMatchSnapshot();
  })
})