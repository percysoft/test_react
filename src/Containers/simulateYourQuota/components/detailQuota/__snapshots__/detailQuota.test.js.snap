// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Item Detail Component Render Item  1`] = `
<styled.div>
  <styled.div
    bold="300"
    size="20"
  >
     
    TEA
  </styled.div>
  <styled.div
    bold="bold"
    size="25"
  >
     
    10000
  </styled.div>
</styled.div>
`;
